jenkins/debian/debos/rootfs.yaml

```diff
@@ -105,6 +107,29 @@ actions:
     chroot: false
     command: cd ${ROOTDIR} ; find -H  |  cpio -H newc -v -o | gzip -c - > ${ARTIFACTDIR}/{{ $basename -}}/full.rootfs.cpio.gz
 
+{{ if $multiple_partition }}
```

Comment:
The partitions should be on the storage device used to run the tests, not in the rootfs image.

Reply:
ah, okay I was under the assumption, its fixed within rootfs.  Will remove this option.